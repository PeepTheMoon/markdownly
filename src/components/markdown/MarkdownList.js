import React from 'react';
import { getMarkdownTitles } from '../../selectors/selectors';
import { useSelector } from '../../hooks/Context';


const MarkdownList = () =>  {

  const markdownList = [
    'test 1',
    'test 2',
    'test 3'
  ];

  const markdownListElements = markdownList.map(({ title, i }) => (
    <li key={title + i}>
      <p>{i}</p>
    </li>
  ));
    
  return (
    <ul>
      {markdownListElements}    
    </ul>
  );
};

export default MarkdownList;
