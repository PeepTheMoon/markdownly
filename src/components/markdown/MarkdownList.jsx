import React from 'react';

const MarkdownList = () =>  {

    
  return (
    <div>
            
    </div>
  );
};

export default MarkdownList;
