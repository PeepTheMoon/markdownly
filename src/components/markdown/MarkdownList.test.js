import React from 'react';
import { shallow } from 'enzyme';
import MarkdownList from './MarkdownList';

jest.mock('../../hooks/Context');

describe('MarkdownList component', () => {
  it('renders MarkdownList', () => {
    const wrapper = shallow(<MarkdownList />);
    expect(wrapper).toMatchSnapshot();
  });
});
