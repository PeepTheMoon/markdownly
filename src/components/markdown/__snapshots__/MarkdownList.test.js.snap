// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarkdownList component renders MarkdownList 1`] = `
<ul>
  <li
    key="NaN"
  >
    <p />
  </li>
  <li
    key="NaN"
  >
    <p />
  </li>
  <li
    key="NaN"
  >
    <p />
  </li>
</ul>
`;
